src/shared/extensionUtilities.ts

```diff
@@ -44,7 +46,11 @@ export function getIdeType(): IDE {
         return IDE.cloud9
     }
 
-    // Theia doesn't necessarily have all env propertie
+    if (vscode.env.appName === sageMakerAppname) {
+        return IDE.sagemaker
+    }
+
+    // Theis doesn't necessarily have all env properties
```

Comment:
```suggestion
    // Theia doesn't necessarily have all env properties
```

this is referring to https://theia-ide.org/

Reply:
updated